deployment/conda/conda_env.yml

```diff
@@ -16,6 +16,9 @@ channels:
 - kitware-danesfield-pt
 - kitware-danesfield-cf
 - kitware-danesfield-df
+- pytorch
+- conda-forge
+- defaults
```

Comment:
I don't recall exactly what else you had needed to do to get this build working, but if these additional channels aren't really needed we should remove them (again to eliminate as many variables as possible)